docs/source/deploy_chaincode.md

```diff
@@ -32,10 +32,24 @@ The `createChannel` command creates a channel named ``mychannel`` with two chann
 We can now use the Peer CLI to deploy the asset-transfer (basic) chaincode to the channel using the following steps:
 
 
-- [Step one: Package the smart contract](#package-the-smart-contract)
-- [Step two: Install the chaincode package](#install-the-chaincode-package)
-- [Step three: Approve a chaincode definition](#approve-a-chaincode-definition)
-- [Step four: Committing the chaincode definition to the channel](#committing-the-chaincode-definition-to-the-channel)
+- [Deploying a smart contract to a channel](#deploying-a-smart-contract-to-a-channel)
```

Comment:
Since this was a tutorial I think Nik's intent here was to list the overall steps for chaincode deployment so people understand the process rather than provide a tutorial toc. Maybe you are thinking of something different here?

Reply:
It also lined up to the chaincode lifecycle key concept doc, so the concept and the tutorial was alligned

Reply:
Not sure why this change got introduced, it seemed to be an automatic TOC type update.  Anyway, I have manually changed it back to the original 4 steps.